.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -40,3 +53,36 @@ body:
         - label: CLI
         - label: Other
 
+  - type: textarea
+    id: steps-to-reproduce
```

Comment:
Does this make sense in the context of a RFE?

Reply:
Yes. We can make it optional, however I don't think we should skip it.